gdbplus/gdb-12.1/gdb/heap_ptmalloc_2_35.c

```diff
@@ -0,0 +1 @@
+../../../src/heap_ptmalloc_2_35.cpp
```

Comment:
Do you think if it is better we seperate the files into two parts: one is specific to gdb version, the other part is immune to the gdb version change?

I have write some code that treat all the code as immune to the gdb version change in https://github.com/yanqi27/core_analyzer/pull/39
I realize later that there are actually two parts. Some code files are specific to the gdb version.

The pros I can see for seperate them as two parts are:
1. We don't have many same files duplicate in the repo. Sometimes it is very hard to locate the file I want to check though I can add a file filter to filter out some files.
2. Symbolic link is not needed anymore. It seems Windows cannot work with symbolic links out of the box and we need to twist something to make it work.
3. We can easily spot our code change from gdb's original code. For example, this PR contains the gdb's `Makefile.in` and we actually change a few lines, but it is hard to spot from the code change.

The cons of having two parts are
1. We need to write a script to copy those two parts into the gdb repo, i.e, we will have our own build script